app/templates/search/macros/filters/subjects_tna.html

```diff
@@ -0,0 +1,57 @@
+{% from 'components/button/macro.html' import tnaButton %}
+{% from 'components/checkboxes/macro.html' import tnaCheckboxes %}
+
+{% macro subjects_tna(request, subjects_tna, number_of_subjects_to_show=10) %}
+  {% set is_tna_results_page = request.GET.get('group') in [None, 'tna'] %}
+  {% if is_tna_results_page %}
+    <div class="tna-aside tna-aside--tight tna-background-tint tna-!--margin-top-s">
+      {% set show_more_link = subjects_tna.items() | length > number_of_subjects_to_show %}
+      {% set subjects = [] %}
+      {% for subject_id, subject in subjects_tna.items() %}
+        {% if loop.index <= number_of_subjects_to_show %}
+          {% set subjects = subjects.append({
+          'text': subject,
+          'value': subject_id,
+          'checked': qs_is_value_active(request.GET, 'subjects', subject_id)
+        }) %}
+        {% elif loop.index > number_of_subjects_to_show %}
+          {% set show_more_link = true %}
+        {% endif %}
+      {% endfor %}
+      {{ tnaCheckboxes({
+            'label': 'Subjects',
+            'headingLevel': 3,
+            'headingSize': 'm',
+            'id': 'subjects',
+            'name': 'subjects',
+            'items': subjects,
+            'small': True,
+            'attributes': {
+              'form': 'search-form'
+            }
+          }) }}
+      <div class="tna-button-group tna-!--margin-top-s">
+        {{ tnaButton({
+              'text': 'Update',
+              'buttonElement': True,
+              'buttonType': 'submit',
+              'small': True,
+              'attributes': {
+                'form': 'search-form'
+              }
+            }) }}
+
+        {% set query_params = request.GET.copy().urlencode() %}
+
+        {% if show_more_link is true %}
+          {{ tnaButton({
+            'text': "See more",
+            'href': "/catalogue/search/?filter_list=subjects&" + query_params,
```

Comment:
I think you should construct this dynamically using Django's `reverse`.

Reply:
Thanks @ahosgood - I hadn't come across Django's `reverse` before. I'm just looking into it now and will amend the commit and push with this change too 👍🏽

Reply:
Do you mind if I push something to this branch? I think it might be easier than trying to explain.

Reply:
Thanks @ahosgood - this looks and works great 😃 I'll make a note to use this approach - and the utilities in `jinja2.py` - for future work 👍🏽 . I've just spotted this branch has a conflict to be resolved so I'm thinking I'll roll the change your commit introduces into the previous commit and then rebase onto an updated `main`, just to keep the three changes in a single commit. Hope that's OK.